### DISCUSSION TITLE
Custom node Handles for each Node is showing edge dots

### DISCUSSION THREAD

**Discussion by dheerajmishradv** [xyflow/xyflow#4016] [Q&A]
Hi all

I need help where I have requirement that I need custom node for each Node. Problem is the <Handle> part which is written in custom node that leads to edge dot visible in source and target Node also. so my requirement is that the <Handle> should be automatically defined using data from initialEdges rather than we defining for customNodes using <handle>

And the other problem I am facing is that if we have <Handle> for source and target  then for that the node borders are not visible

**Comment by bcakmakoglu** [comment]
I'm not sure if I understand what you're trying to express.
You do not want handles in your custom nodes? That's not gonna work. 
You *need* handles for nodes+edges to work and the library isn't going to auto-generate handles for you.

Maybe you're looking for something like this though: https://reactflow.dev/examples/edges/floating-edges
